tools/flake8.py

```diff
@@ -0,0 +1,12 @@
+"""Wrapper around flake8.
+
+Exists, so that we can define a Bazel py_binary target that combines flake8 and
+any flake8 plugins we might need.
+
+"""
+import flake8.main.cli
+
+import sys
+
+if __name__ == '__main__':
+  sys.exit(flake8.main.cli.main())
```

Comment:
you seem to be having intendation issues here 8/

Reply:
argh - just realized the rest has google indents

**request changes!!!**

Reply:
Ignoring and walking away slightly entertained.